test/powershell/Modules/Microsoft.WSMan.Management/CredSSP.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+Describe "CredSSP cmdlet tests" -Tags 'Feature','RequireAdminOnWindows' {

```

Comment:
Just tried it combined and `start-pspester` only ran the tests as elevated so that one test gets skipped.  I'll have to have the non-admin tests as separate.